src/eyepop-render-2d/render-box.ts

```diff
@@ -21,16 +21,18 @@ export class RenderBox implements Render
     private showNestedClasses: boolean
     private showConfidence: boolean
     private showTraceId: boolean
+    private boxType?: string
 
     constructor(options: Partial<RenderBoxOptions> = {})
     {
-        const { showClass = true, showConfidence = false, showTraceId = false, showNestedClasses = false, target = '$..objects.*' } = options
+        const { showClass = true, showConfidence = false, showTraceId = false, showNestedClasses = false, target = '$..objects.*', boxType = "rich" } = options
```

Comment:
We should define the `boxType` as enum

Reply:
Added 👍

Reply:
<img width="1163" alt="image" src="https://github.com/user-attachments/assets/f5dd52d0-921e-44b7-8b59-ce2ba068ca73">